contributors/devel/sig-testing/writing-good-conformance-tests.md

```diff
@@ -0,0 +1,56 @@
+# Writing Good Conformance Tests for Kubernetes #
+
+Conformance tests are promoted from existing e2e tests which have
+been proven to be stable and "non-flaky".
+
+Please read the the 
+["Writing good e2e
+tests"](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/writing-good-e2e-tests.md)
+guide as a supplment to this document.
+
+## Patterns and Anti-Patterns ##
```

Comment:
this organizational structure seems weird to me.

Reply:
I based the structure on ["Writing Good e2e Tests"](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/writing-good-e2e-tests.md) doc. I agree. I don't think it fits as well here.

Reply:
Removed "Patterns and Anti-Patterns" sub-title in https://github.com/kubernetes/community/pull/4107/commits/1e36e88c5fda6792107b3fe95316ef226f597f35